doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -48,8 +48,14 @@ Build and configuration system
 Bootloaders and DFU
 ===================
 
-* Added documentation for :ref:`qspi_xip_split_image` functionality.
-* Added a section in the sysbuild-related migration guide about the migration of :ref:`child_parent_to_sysbuild_migration_qspi_xip` from child/parent image to sysbuild.
+* Added: 
+  
+  * Documentation for :ref:`qspi_xip_split_image` functionality.
+  * A section in the sysbuild-related migration guide about the migration of :ref:`child_parent_to_sysbuild_migration_qspi_xip` from child/parent image to sysbuild.
+ 
+* Updated the procedure for signing the application image built for booting by MCUboot in directXIP mode with revert support.
```

Comment:
Should "directXIP" be written as "DirectXIP"?

Reply:
I've been wondering about it and don't have a 100% answer, mostly because we have yet another spelling we use in our documentation (most commonly direct-xip). In Zephyr documentation and other external pages it is, in fact, commonly written as DirectXIP, capitalized.

Reply:
It seems in the doc https://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-17419/nrf/app_dev/device_guides/nrf52/fota_update.html, "direct-xip" is consistently used. To align with our documentation, it would be advisable to maintain consistency.

I've noticed two different styles being used in this PR, hence pointed out.

Reply:
It's either align it with our documentation or update all other occurrences, so agreed, let's align it for now. Added once again the initial suggestions